docker-for-windows/release-notes.md

```diff
@@ -13,10 +13,43 @@ This page contains information about the new features, improvements, known issue
 
 {% include eula.md %}
 
+## Docker Desktop 3.5.0
+2021-06-23
+
+[Download](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64){: .button .primary-btn}
+
+### New
+
+**Dev Environments Preview**: Dev Environments enable you to seamlessly collaborate with your team members without moving between Git branches to get your code onto your team members' machines. When using Dev Environments, you can share your in-progress work with your team members in just one click, and without having to deal with any merge conflicts. For more information and for instructions on how to use Dev Environments, see [Development Environments Preview](../desktop/dev-environments.md).
+
+### Upgrades
+
+- [Compose V2 beta](https://github.com/docker/compose-cli/releases/tag/v2.0.0-beta.4)
+  - Fixed a bug where a container cannot be started when a file is bind-mounted into a nested mountpoint. Fixes [docker/compose-cli#1795](https://github.com/docker/compose-cli/issues/1795).
+  - Added support for container links and external links.
+  - Introduce `docker compose logs --since --until` option.
+  - `docker compose config --profiles` now lists all defined profiles.
+- From [Kubernetes 1.21.1](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.1) to [Kubernetes 1.21.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.21.2)
+
+
+### Bug fixes and minor changes
+
+- **Volume Management**
+  - Able to remove a file or directory from the Docker Dashboard.
+  - Show more last time modified and size of the contents in the volumes.
+  - The files inside the volume can be saved form the Docker Dashboard.
+- Fixed an issue that caused credStore timeout errors when running the `docker login` command. Fixes [docker/for-win#11472](https://github.com/docker/for-win/issues/11472)
+- Docker Desktop now allows the WSL 2 integration agent to start even when `/etc/wsl.conf` is malformed.
+- Fixed an issue with the Docker Compose app not being stopped or removed when started by multiple configuration files. [docker/for-win#11445](https://github.com/docker/for-win/issues/11445)
+- Fixed a bug where Docker Desktop fails to restart after a power failure because the Hyper-V VM restarted prematurely.
+- Fixed a link to the policy that provides details on how Docker handles the uploaded diagnostics data. Fixes [docker/for-mac#5741](https://github.com/docker/for-mac/issues/5741)
+- The default Windows named pipe name is now `\\.\pipe\dockerDesktopWindowsEngine` and the Linux named pipe name is `\\.\pipe\dockerDesktopLinuxEngine`. For backwards compatibility `\\.pipe\docker_engine` is still available. We recommend using the `docker context`s called `desktop-linux` and `desktop-windows`.
```

Comment:
Similarly to the Mac case above, perhaps this one change should become:

- The default `docker` CLI `context` is now `desktop-linux` in Linux containers mode and `desktop-windows` when in Windows containers mode.
- The Linux engine Docker API is now available on `\\.\pipe\dockerDesktopLinuxEngine`.
- The Windows engine Docker API is now available on `\\.\pipe\dockerDesktopWindowsEngine`.

Reply:
Is this a thing we want to advertise to the user though? Shouldn't it be a black box?